pkg/authorize/ssl/ssl.go

```diff
@@ -1,11 +1,19 @@
 package ssl
 
 import (
+	"context"
 	"errors"
 	"fmt"
 	"net/http"
+	"strings"
 
 	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+const (
+	OrganizationContextKey = "org"
+	CommonNameContextKey   = "common_name"
```

Comment:
Not a good practice to use strings as context keys. We can use something like `struct{}` (empty struct) to avoid allocations, among other options.

Reply:
Reference: https://pkg.go.dev/context#WithValue